### DISCUSSION TITLE
Blanket import

### DISCUSSION THREAD

**Discussion by kdwk** [Relm4/Relm4#514] [Ideas]
When trying SwiftUI, I noticed that I can put `import SwiftUI` and everything would be imported, no need to specify which component. I think this would greatly improve developer experience for newcomers, who may not be familiar with all the different components and would frequently run into the problem where they don't know which crate they need to import what they want from.

<img width="1136" alt="Screenshot 2023-08-27 at 10 19 32 PM" src="https://github.com/Relm4/Relm4/assets/39268094/d67a5df0-28cd-4f04-ba32-d398deb302de">

Perhaps the Relm4 could be something like:

<img width="798" alt="Screenshot 2023-08-27 at 10 42 26 PM" src="https://github.com/Relm4/Relm4/assets/39268094/fecb3585-072f-4447-a783-09710e7da77b">

**Comment by kdwk** [comment]
Perhaps #[relm4::component(libadwaita)] can be shortened to #[relm4::component(adw)] too

**Comment by AaronErhardt** [comment]
There is a prelude in the gtk4, libadwaita and relm4 crate that you can use for most important types and especially traits.

So, albeit a bit longer than the SwiftUI import, you could use something like this:

```rust
use gtk::{*, prelude::*};
use adw::{*, prelude::*};
use relm4::{*, prelude::*};
```
